### DISCUSSION TITLE
Conda Installation for esmpy 8.6 fails on Windows

### DISCUSSION THREAD

**Discussion by emanuel-schmid** [esmf-org/esmf#260] [Building / Installing] [answered]
### Requirements

- [X] Reviewed [ESMF User's Guide](https://earthsystemmodeling.org/doc/)
- [X] Searched [GitHub Discussions](https://github.com/orgs/esmf-org/discussions?discussions_q=)

### Affiliation(s)

ETH Zurich

### ESMF Version

v8.4.1

### Issue

I have tried to install esmpy 8.6 (or any version other than 8.4) on Windows via mamba.
It fails with the following error message

```bash
> mamba install esmpy=8.6 -c conda-forge
...
Could not solve for environment specs
The following packages are incompatible
└─ esmpy 8.6**  is uninstallable because there are no viable options
   ├─ esmpy 8.6.0 would require
   │  └─ esmf 8.6.0.* , which does not exist (perhaps a missing channel);
   └─ esmpy 8.6.1 would require
      └─ esmf 8.6.1.* , which does not exist (perhaps a missing channel).
```
Is this something that can fixed by me or something you will fix at some point or is there another way to install emspy 8.6 on Windows?
(I'm aware that 8.4 is available for Windows, but 8.4 is not compatible with the latest importlib-metadata.)
(Installation via command line is mandatory in my case.)

Many thanks for your support!

### Autotag

@oehmke

**Comment by billsacks** [comment, marked as answer]
The conda build of ESMF on Windows was abandoned for more recent versions as noted here: https://github.com/conda-forge/esmf-feedstock/issues/115, due to a compiler bug in the old version of gfortran used in that build chain.

You may be able to install ESMF from source - see https://earthsystemmodeling.org/docs/release/latest/ESMF_usrdoc/node10.html (particularly see Cygwin and MinGW options for ESMF_OS).

Alternatively, if you're able to use a Docker container for this work, you could use the ESMF Docker image, which contains ESMPy as well as ESMF:

```
docker pull esmf/esmf-build-release:latest
docker run -it esmf/esmf-build-release:latest
```

**Reply by emanuel-schmid** [reply]
thanks for clarifying!
